sonic_installer/main.py

```diff
@@ -129,6 +129,14 @@ def get_docker_tag_name(image):
     return tag
 
 
+def write_msg_cli_syslog(msg, msg_type="notice"):
```

Comment:
>msg_type="notice" [](start = 30, length = 17)

Could you mimic the click.echo paramters, such as `err`

Reply:
Suggest changing name of function to something like `echo_and_log()`

Also, instead of passing a string with the log level, suggest passing constants from sonic_py_common: https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py#L20-L24

Then you don't need the branches below. You can instead call `log.log()` and pass the parameter. See here: https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py#L92